### ISSUE TITLE
[Bug] `zebar -V` doesn't return the correct version on Windows

### ISSUE THREAD

**Issue by azinsharaf** [glzr-io/zebar#51]
I have used [Zebar_x64_v1.3.0.msi](https://github.com/glzr-io/zebar/releases/download/v1.3.0/Zebar_x64_v1.3.0.msi) file to install it on Windows 11 Pro. The binary verison is being showed as `0.0.0`. 

```
 >  zebar -V
desktop 0.0.0

 >  zebar --version
desktop 0.0.0
```

**Comment by Doom-Git** [comment]
I think the problem is, that the version does get pulled from tauri.conf.json. The version their is "0.0.0". 
![image](https://github.com/user-attachments/assets/2a16726b-3db3-4e10-815b-4bfc761bf0ad)

**Comment by lars-berger** [comment]
Fixed in v2.5.0 👌
